metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1672,6 +1675,23 @@ class MetalsLanguageServer(
     results
   }
 
+  /** Requesting semantic tokens for a whole file in order to highlight */
+  @JsonRequest("textDocument/semanticTokens/full")
+  def semanticTokensFull(
+      params: SemanticTokensParams
+  ): CompletableFuture[SemanticTokens] = {
+    scribe.info("Debug: MetalsLanguageServer.semanticHighlighting: Start")
+
+    CancelTokens.future { token =>
+      compilers.semanticTokens(
+        params,
+        SemanticTokenCapability.TokenTypes,
```

Comment:
I think both `TokenTypes` and `TokenModifiers` will be the same within the server lifetime, so we can remove them as parameters and just use them directly.